pkg/baseline/packageIndex.json

```diff
@@ -1,3 +1,7 @@
+// StableVersions: Released versions.
+// BaselineVersion: Used to determine the minimum version used for all packages in the repo.
+// AssemblyVersionInPackageVersion: Used to understand what version of a package to reference.
+//                                  Then raise the package version of the reference so that its at least the baseline version.
```

Comment:
So long as our JSON parser is ok with this I am fine.  Another pattern folks use is to create properties in the json.

EG:
```json
  "__comment" : [
    "StableVersions: Released versions.",
    "BaselineVersion: Used to determine the minimum version used for all packages in the repo."
    ...
  ]